mlir/python/QuantumExtension.cpp

```diff
@@ -48,4 +48,17 @@ PYBIND11_MODULE(_quantumDialects, m)
             }
         },
         py::arg("context") = py::none(), py::arg("load") = true);
+
+    auto catalyst_m = m.def_submodule("catalyst");
+
+    catalyst_m.def(
+        "register_dialect",
```

Comment:
Would you like to add this to changelog for the future reference?

Reply:
https://github.com/PennyLaneAI/catalyst/pull/345/commits/43216a3e2c331d5920be11e6b2b0017f6b0a757c